frontend/kesaseteli/employer/src/components/application/steps/step2/accordions/EmploymentAccordionHeader.tsx

```diff
@@ -0,0 +1,38 @@
+import { IconAlertCircleFill } from 'hds-react';
```

Comment:
represents accordion header:
<img width="971" alt="Screenshot 2021-09-20 at 3 53 21 PM" src="https://user-images.githubusercontent.com/7648194/134005527-78e35dfa-53c9-4186-8f55-a5c37bab1b28.png">
It indicates error when accordion is closed and form inside is invalid